src/main/java/com/github/exadmin/sourcesscanner/exclude/ExcludeSignatureItem.java

```diff
@@ -32,6 +32,9 @@ public void setFileHash(String fileHash) {
         this.fileHash = fileHash;
     }
 
+    /**
+    * Override equals method for the ExcludeSegnatureItem-class
```

Comment:
Typo in class name 'ExcludeSegnatureItem' should be 'ExcludeSignatureItem'.

```suggestion
* Override equals method for the ExcludeSignatureItem-class
```

#deepseek-review:inline